geomesa-index-api/src/main/scala/org/locationtech/geomesa/index/utils/KryoLazyStatsUtils.scala

```diff
@@ -0,0 +1,75 @@
+package org.locationtech.geomesa.index.utils
+
+import org.apache.commons.codec.binary.Base64
+import org.geotools.factory.Hints
+import org.locationtech.geomesa.features.ScalaSimpleFeature
+import org.locationtech.geomesa.index.conf.QueryHints.{ENCODE_STATS, RichHints, STATS_STRING}
+import org.locationtech.geomesa.utils.collection.CloseableIterator
+import org.locationtech.geomesa.utils.geotools.GeometryUtils
+import org.locationtech.geomesa.utils.interop.SimpleFeatureTypes
+import org.locationtech.geomesa.utils.stats.{Stat, StatSerializer}
+import org.opengis.feature.simple.{SimpleFeature, SimpleFeatureType}
+
+/**
+  * Created by Spandan on 5/8/17.
+  */
```

Comment:
Can you remove this header and add the usual license header?  

You can test the headers locally with 'mvn license:check'.